PROCESSORS.md

```diff
@@ -971,13 +971,13 @@ In the list below, the names of required properties appear in bold. Any other pr
 |**Known Brokers**|||A comma-separated list of known Kafka Brokers in the format <host>:<port><br/>**Supports Expression Language: true**|
 |Max Flow Segment Size|0 B||Maximum flow content payload segment size for the kafka record. 0 B means unlimited.|
 |Max Request Size|||Maximum Kafka protocol request message size|
-|Message Key Field|||The name of a field in the Input Records that should be used as the Key for the Kafka message.
-Supports Expression Language: true (will be evaluated using flow file attributes)|
-|Message Timeout|30 sec||The total time sending a message could take<br/>**Supports Expression Language: true**|
+|Kafka Key|||The key to use for the message. If not specified, the UUID of the flow file is used as the message key.<br/>**Supports Expression Language: true**|
```

Comment:
nitpick: other properties seem to be in an alphabetical order

Reply:
fixed